hr_attendance_schedule/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Attending Schedule
+==================
+
+This module extends the functionality of Attending to allow schedules
+to round checkins and checkouts.
+
+It also synchronizes the clock in Kiosk mode with the server.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to *Attendance > Manage Attendances > Schedules*
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/hr/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
```

Comment:
```suggestion
Authors
~~~~~~~

* Métal Sartigan

```

Reply:
I took a look at another README.rst and it seems to be a generated file which uses files in a readme folder. How does it work? There's nothing about it in the [contribution doc](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst)

Reply:
Ok I found some info [here](https://github.com/OCA/maintainer-tools#readme-generator).